src/cmd/root.go

```diff
@@ -174,6 +258,9 @@ func init() {
 
 	vpr := getViper()
 
+	// Features
+	rootCmd.PersistentFlags().StringVar(&features, "features", vpr.GetString(VFeatures), "[ALPHA] Provide a comma-separated list of feature names to bools to enable or disable. Ex. --features \"foo=true,bar=false,baz=true\"")
```

Comment:
nit: Should we use StringToStringVar to make parsing slightly simpler

Reply:
Given that we have other stringToString variables that have seen significant use such as `[package.create.set]`, I'd like to see us stick with that same format for the viper config files. That said, I agree that the default way that viper decodes these environment variables (a json string) is a very poor UX. I think a custom parser could work. Though I also believe that the CLI flag and config file are sufficient for now, and if we get user feedback that they'd like to use environment variables we can add that later.